src/genie/libs/parser/hp_comware/tests/DisplayInterfaces/cli/equal/golden_output1_expected.py

```diff
@@ -0,0 +1,86 @@
+expected_output = {
+    "GigabitEthernet3/8/0/48": {
+        "auto_negotiate": True,
+        "counters": {
+            "in_abort": 0,
+            "in_broadcast_pkts": 0,
+            "in_crc_errors": 0,
+            "in_errors": 0,
+            "in_frame": 0,
+            "in_giants": 0,
+            "in_ignored": 0,
+            "in_mac_pause_frames": 0,
+            "in_multicast_pkts": 3832415,
+            "in_octets": 4508060351380,
+            "in_overrun": 0,
+            "in_parity_errors": 0,
+            "in_pkts": 3184372634,
```

Comment:
no entirely sure what the difference is between (total) and (normal).
```
 Input (total):  3184372634 packets, 4508060351380 bytes
         3180540219 unicasts, 0 broadcasts, 3832415 multicasts, 0 pauses
 Input (normal):  3184372634 packets, - bytes
         3180540219 unicasts, 0 broadcasts, 3832415 multicasts, 0 pauses
```
no need to gather both separately?

Reply:
I think the best should be gather separately, going to modify the code.

Reply:
Total: All inbound normal packets, abnormal packets, and normal pause frames were counted.
Normal the normal traffic.
Maybe change the schema to somethink like that:
counters : {
  normal : {
    "in_octets":  int,
    "in_overrun" : int,
  }
  "in_octets":  int,
  "in_overrun" : int,
}
The keys under counters means the Total and the ones inside normal the normal, what do you think?

Reply:
I like your idea. trying to keep similar schema with other parsers.

Reply:
Fixed in 90e527d